### ISSUE TITLE
Exception when using Sequel rather than AR

### ISSUE THREAD

**Issue by ericgross** [voormedia/flipflop#1]
I'm not using ActiveRecord, but rather Sequel. I have implemented a `SequelStrategy` which I'd be happy to share. I'd be fine simply disabling the Flipflop::Feature class totally or just removing the AR dependency, whichever is easier. I'm not even using this class.

```
2016-05-04T00:25:31.445373+00:00 app[web.1]: [3] ! Unable to load application: NameError: uninitialized constant ActiveRecord
2016-05-04T00:25:31.446144+00:00 app[web.1]: /app/vendor/bundle/ruby/2.3.0/gems/flipflop-2.2.0/app/models/flipflop/feature.rb:1:in `<top (required)>': uninitialized constant ActiveRecord (NameError)

```

**Comment by rolftimmermans** [comment]
Thanks for letting me know! Should be fixed in 8f0aebf958 (and in 2.2.1 soon).

**Comment by rolftimmermans** [comment]
Also, feel free to open a pull request to include the `Sequel` strategy!
